cves/CVE-2014-0050.yml

```diff
@@ -118,7 +130,7 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: Searched through each link on cve.mitre.org as well as a thorough search of the git log and found nothing on the origin of this bug.
```

Comment:
Hi Nick, if you check the security bulletin on the Apache Struts website you may find some information on who discovered the vulnerability.

Reply:
Hey Matt, do you have more information on where to look? Possibly a link? I can't seem to find anything.

Reply:
Doesn't have too much information, but it has who discovered it. https://cwiki.apache.org/confluence/display/WW/S2-020

Reply:
Thanks, I added his name in there.